src/components/CardsContainer.tsx

```diff
@@ -1,68 +1,78 @@
 'use client'
 
 import UserCard from './UserCard'
+import UserShowPointsCard from './UserShowPointsCard'
 import { useState } from 'react'
 
-import { namesArray } from '@/utils/sampleData'
-import { sealifeImages } from '@/utils/imageLists'
 import { useSocketListener } from '@/services/socket'
 import type { ListenerRes } from '@/services/socket'
 
-type CardUserPoint = {
-	userName: string
-	storyPoints: string
-	imageSrc: string
-}
-
 export default function CardsContainer() {
-	// TODO: Need a sorting function to sort the users by points.
-	// Need to map usersPointsData to CardUserPoint.
-	// Need animation for bring cards into high-low order by points when showStoryPoints is true.
-	const allUsersStoryPoints = useSocketListener('all-users-story-points')
+	const [sortedUsersPoints, setSortedUsersPoints] = useState<ListenerRes[]>([])
+
+	const allUsersStoryPoints = useSocketListener('all-users-story-points', {
+		onChange: (allPointsRes) => {
+			const allUsersPointsSorted = [...(allPointsRes.message as unknown as ListenerRes[])]
+
+			allUsersPointsSorted.sort((a, b) => {
+				const aMessage = a.message as unknown as number
```

Comment:
> "i'm gonna trust JS with numbers, but not strings"

thought: you shouldn't trust either, but i think i trust strings more than numbers. 0.3 + 0.1 is a nightmare hah

Reply:
I shall amend my statement:  I trust JS with integers more than strings, and strings more than floats. I think that's a statement I can defend. 

What I was actually thinking was that I trust JS with integers ONLY or strings ONLY. I didn't like the fact that I was mixing strings and numbers a bit too much. And of course, there are calculations that need numbers (integers!!).